README.txt

```diff
@@ -27,8 +27,9 @@ The configuration settings are:
   username - The livejournal user name. A subdirectory will be created
              with this same name to store the journal entries.
 
-  password - The account password. This password is never sent in the
-             clear; the livejournal "challenge" password mechanism is used.
+  password - The account password. This password is sent in the clear,
+             so if you specify an alternative server, ensure you use
+             a URL starting with https:// so the connection is encrypted.
```

Comment:
Looking back at this PR, I wonder if plain-HTTP should be rejected at this point -- or at least require the user to be prompted for approval.